OpenSSL/crypto.py

```diff
@@ -2304,6 +2372,19 @@ def verify(cert, signature, data, digest):
         _raise_current_error()
 
 
+def verify_cert(store_ctx):
+    """
+    Verify a certificate in a context.
+
+    :param store_ctx: The :py:class:`X509StoreContext` to verify.
+    :raises: Error
+    """
+    store_ctx._init()
+    ret = _lib.X509_verify_cert(store_ctx._store_ctx)
+    store_ctx._cleanup()
```

Comment:
I think that is how we originally thinking about implementing this but stopped once we realized that this is _not_ how the actual OpenSSL API is exposed. That is, the OpenSSL API has this `verify_cert` method as a top-level function.

With that said, I'm happy to move it to see how it looks, just wanted to remind us why we chose to do it this way a couple months ago. :)